app/assets/javascripts/visualizations/highvis/map.coffee

```diff
@@ -173,6 +173,32 @@ $ ->
             label  = "<div style='font-size:9pt;overflow-x:none;'>"
             label += "<div style='width:100%;text-align:center;color:#{color};'> " +
               "#{dataPoint[globals.configs.groupById]}</div>"#<br>"
+
+            if data.metadata[0].photos.length == 1
+              photo = data.metadata[0].photos[0]
+              label += "<img src=#{photo.src} style='width:100%; height:300px;'>"
+
+            else if data.metadata[0].photos.length > 0
+              label += "<div id='mapCarousel' class='carousel slide' data-ride='carousel' data-interval='false'>
+                        <div class='carousel-inner' role='listbox'>"
+
+              firstPhoto = data.metadata[0].photos[0]
+              label += "<div class='item active'>
+                          <img src=#{firstPhoto.src} style='width:100%; height:300px;'>
+                        </div>"
+
+              for i in [1...data.metadata[0].photos.length]
+                label +=  "<div class='item'>
+                            <img src=#{data.metadata[0].photos[i].src} style='width:100%; height:300px;'>
```

Comment:
No inline styles. Just add a class instead. There might already be one with these properties.